nixos/modules/services/web-apps/dawarich.nix

```diff
@@ -0,0 +1,744 @@
+{
+  lib,
+  pkgs,
+  config,
+  options,
+  ...
+}:
+
+let
+  cfg = config.services.dawarich;
+  opt = options.services.dawarich;
+
+  # We only want to create a Redis and PostgreSQL databases if we're actually going to connect to it local.
+  redisActuallyCreateLocally =
+    cfg.redis.createLocally && (cfg.redis.host == "127.0.0.1" || cfg.redis.enableUnixSocket);
+  databaseActuallyCreateLocally =
+    cfg.database.createLocally && cfg.database.host == "/run/postgresql";
+
+  env = {
+    RAILS_ENV = "production";
+    NODE_ENV = "production";
+
+    SELF_HOSTED = "true";
+    STORE_GEODATA = "true";
+    APPLICATION_PROTOCOL = "http";
+    TIME_ZONE = config.time.timeZone; # otherwise upstream forces it to Europe/London
+    DOMAIN = cfg.localDomain;
```

Comment:
Shouldn't this set `APPLICATION_HOSTS` instead of `DOMAIN`? Considering the docs: 
https://dawarich.app/docs/environment-variables-and-settings/

(Trying this PR without setting  `APPLICATION_HOSTS` didn't work for me.)